daml-script/export/integration-tests/export-lf-1.6/BUILD.bazel

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+load(
+    "//bazel_tools/client_server:client_server_build.bzl",
+    "client_server_build",
+)
+load(
+    "//bazel_tools:scala.bzl",
+    "da_scala_binary",
+)
+load(
+    "//bazel_tools/sh:sh.bzl",
+    "sh_inline_test",
+)
+load(
+    "//rules_daml:daml.bzl",
+    "daml_compile",
+)
+
+daml_compile(
+    name = "lf16",
+    srcs = ["daml/LF16.daml"],
+    target = "1.6",
+    version = "1.0.0",
+)
+
+da_scala_binary(
+    name = "lf16-export-client",
+    srcs = ["scala/com/daml/script/export/LF16ExportClient.scala"],
+    main_class = "com.daml.script.export.LF16ExportClient",
+    scala_deps = [
+        "@maven//:com_github_scopt_scopt",
+    ],
+    deps = [
+        "//daml-lf/archive:daml_lf_archive_reader",
+        "//daml-lf/data",
+        "//daml-lf/language",
+        "//daml-script/export",
+        "//language-support/scala/bindings",
+        "//language-support/scala/bindings-akka",
+        "//ledger-api/rs-grpc-bridge",
+        "//ledger/ledger-api-common",
+        "//libs-scala/auth-utils",
+        "//libs-scala/fs-utils",
+    ],
+)
+
+client_server_build(
+    name = "lf16-export",
+    outs = [
+        "lf16-export.zip",
+    ],
+    client = ":lf16-export-client",
+    client_args = [
+        "--ledgerid",
+        "lf16-export-ledger",
+    ],
+    client_files = ["lf16.build"],
+    data = ["lf16.build"],
+    output_env = "EXPORT_OUT",
+    server = "//ledger/sandbox:sandbox-binary",
+    server_args = [
+        "--port",
+        "0",
+        "--ledgerid",
+        "lf16-export-ledger",
+    ],
+    server_files = ["lf16.build"],
+)
+
+sh_inline_test(
+    name = "lf16-export-build",
+    cmd = """\
+set -eoux pipefail
+tmpdir=$$(mktemp -d)
+trap "rm -rf $$tmpdir" EXIT
+DAMLC=$$(canonicalize_rlocation $(rootpath //compiler/damlc))
+unzip $$(canonicalize_rlocation $(rootpath :lf16-export.zip)) -d $$tmpdir
+cp -L $$(canonicalize_rlocation $(rootpath //daml-script/daml:daml-script-1.7.dar)) $$tmpdir/
+sed -i.bak 's/daml-script/daml-script-1.7.dar/' $$tmpdir/daml.yaml
+DAML_PROJECT=$$tmpdir $$DAMLC build
```

Comment:
This does not test whether the generated script produces an equivalent ledger state when executed. I think it could be extended to test this as well. However, this PR seemed already big enough as is. So, this is perhaps best left for a follow-up if deemed necessary.

Alternatively, we could just add a golden test for the generated export script.